src/Service/Startup.cs

```diff
@@ -43,6 +43,7 @@ public class Startup
         public static bool IsLogLevelOverriddenByCli;
 
         public const string NO_HTTPS_REDIRECT_FLAG = "--no-https-redirect";
+        public const string BANANA_CAKE_POP_DISABLED_ERROR_CODE = "HC0013";
```

Comment:
This error will occur if a GET request is made without any query or say a POST request is made with no body.

Reply:
I am getting this error when i do empty POST request  "message": "There are no operations in the GraphQL document.",

Reply:
The following is expected because incomplete payload/metadata is provided with your request
> i do empty POST request "message"

Reply:
`The following is expected because incomplete payload/metadata is provided with your request`

Agree, this is good, I just brought this up as it's mentioned above as `say a POST request is made with no body.` That's not the case I think